chat-client-ui-types/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@aws/chat-client-ui-types",
-    "version": "0.0.3",
+    "version": "0.1.0",
```

Comment:
Why are we doing a new minor version rather than a new patch version? 

The changed folder structure shouldn't change how the package is consumed

Reply:
Because now the `out/` package is being published as well, the imports have to be updated to `@aws/chat-client-ui-types/out`

Reply:
The out package is being published but consumers won't have to update their imports because the entry point in package.json points to `out/index.js`  https://github.com/aws/language-server-runtimes/blob/main/chat-client-ui-types/package.json#L5 so imports from `@aws/chat-client-ui-types` should work as before

Reply:
Though, this is misconfigured right now: https://github.com/aws/language-server-runtimes/blob/main/chat-client-ui-types/package.json#L5

We need to change `/out/index.js` to `out/index.js`

Reply:
Imports did not work because of typo in main entry, fixing it in https://github.com/aws/language-server-runtimes/pull/128

I'll close this PR in favour of a new one